src/Components/Pages/SetWorkLoadPage.js

```diff
@@ -109,59 +99,80 @@ const SetWorkloadPage = ({
               }
               position="left"
             >
-              {button}
+              <Radio
+                label={wl}
+                onChange={() => setWorkload(wl)}
+                isChecked={wl === workload}
+                className="pf-v5-u-mb-md"
+                name={wl}
+                id={wl}
+                isDisabled={isDisabled}
+              />
```

Comment:
Issue: Something is wrong with this component - running locally it defaults to "Latest Release" even when I am editing an EUS key